modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -1994,8 +1994,6 @@ def sort_rows(self, columns, ascending, ignore_index, na_position):
                 drop_index_cols_after = [
                     col for col in base._index_cols if col in columns
                 ]
-                if not drop_index_cols_after:
-                    drop_index_cols_after = None
```

Comment:
This variable cannot be None, since further on in the code we are trying to iterate over it.